db/seeds.rb

```diff
@@ -125,8 +125,8 @@
 end.save!
 
 TrafficCam.find_or_initialize_by(
-  title: "Tobey Street"
+  title: "Dean Street"
 ) do |r|
-  r.url = "http://www.dot.ri.gov/img/travel/camimages/6-2%20Rt%206%20E%20%20%2010%20N%20@%20Tobey%20St.jpg"
+  r.url = "https://www.dot.ri.gov/img/travel/camimages/6-1%20Rt%206%20E%20%20%2010%20N%20@%20Dean%20St.jpg"
```

Comment:
What would be the purpose of changing http to https for the link?

Reply:
It does not really matter. So far I know, https is more secure than http but either of them works to open the link.

Reply:
You should basically always prefer `https` to `http` when you have the choice, even if you don't directly care about any of the security features that TLS gives you.

Pages served to the user via HTTP will happily download HTTPS assets. However, pages served over HTTPS will refuse to fetch HTTP assets as doing so could compromise the security guarantees HTTPS makes for the page as a whole.